src/geo/leaflet/leaflet-cartodb-layer-group-view.js

```diff
@@ -101,11 +101,6 @@ LeafletCartoDBLayerGroupView.prototype = _.extend(
           this._onFeatureOver(latlng, containerPoint, waxEvent.data, waxEvent.layer);
           break;
         case 'click':
-        case 'touchend':
```

Comment:
We want to handle the click event, so it is kind of smell when we are mixing events like pointermove on this, also the benefits of using touch-* or pointer-* is to have multitouch, control pointer pressure, etc... that we are not using.

Here, we only want to track the click event AFAIK, maybe in the past we needed them to browser compatibility or mobile support, but now they are carrying on with problems, for example... the click event is not fired when a touchmove is fired between the touchstart/pointerdown and touchend/pointerend, so rare things start to happens.

The problem comes to life when, apparently, leaflet doesn't stop some events propagations, this makes Leaflet more powerful allowing us to make things with those events but exposes problems like open the infowindow each panning of the map.

Maybe I am missing something, any kind of trick that I am not considering here.

What do you think?